articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,88 @@
+# :name_badge: Semantic HTML Elements
+
+For this article, I assume you have a basic working knowledge of HTML. If this is not the case, please [email me](mailto:maddie.horowitz@addepar.com) and I will bring you up to speed.
```

Comment:
Not sure this intro comment is even needed, since this article isn't covering super advanced info and should be pretty easy to understand if you are just learning. I think it's fine though to link to another one of your tutorials.